app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -85,10 +121,26 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
   }
 
   render() {
+    const options = this.createOptionsFromKeys()
+    const { loadingPublicKeys } = this.state
     return (
       <div id="loginLedgerNanoS" className={styles.flexContainer}>
         <form>
           {this.renderStatus()}
+          <div className={styles.publicAddressLabelContainer}>
+            <Label label="public address:" />
+            {loadingPublicKeys && (
+              <RefreshIcon className={styles.ledgerStageRefreshIcon} />
+            )}
+          </div>
+
+          <StyledReactSelect
+            value={this.state.address}
+            onChange={address => this.setState({ address })}
+            options={options}
+            onMenuScrollToBottom={() => this.fetchAdditionalKeys()}
```

Comment:
if `fetchAdditionalKeys` was an arrow function, this we could just write `onMenuScrollToBottom={this.fetchAdditionalKeys}`